packages/migrate/migrations/package/migrate_config.spec.js

```diff
@@ -0,0 +1,83 @@
+import { test } from 'uvu';
+import * as assert from 'uvu/assert';
+import { remove_package_from_config } from './migrate_config.js';
+
+test('Removes package config #1', () => {
+	const result = remove_package_from_config(`
+    export default {
+        kit: {
+            files: {
+                lib: 'src/lib'
+            }
+        },
+        package: {
+            dir: 'package',
+            exports: (filepath) => !/^_|\/_|\.d\.ts$/.test(filepath),
+            files: () => true
+        },
+        preprocess: []
+    }`);
+	assert.equal(
+		result,
+		`
+    export default {
+        kit: {
+            files: {
+                lib: 'src/lib'
+            }
+        },
+        preprocess: []
+    }`
+	);
+});
+
+test('Removes package config #2', () => {
+	const result = remove_package_from_config(`
+    export default {
+        package: {
+            dir: 'package',
+            exports: (filepath) => !/^_|\/_|\.d\.ts$/.test(filepath),
+            files: () => true
+        },
+    }`);
+	assert.equal(
+		result,
+		`
+    export default {
+        }`
```

Comment:
might be worth special-casing objects with a single `package` property to avoid the weird formatting

```suggestion
    export default {}`
```